examples/xml-to-record-conversion/xml_to_record_conversion.out

```diff
@@ -0,0 +1,2 @@
+bal run xml_to_record_conversion.bal
```

Comment:
```suggestion
bal run xml_to_record_conversion.bal
```
Same as above.

Reply:
Sorry, we need not add the $sign on the master branch as it is available only on the new website.